packages/client-mav/package.json

```diff
@@ -12,101 +12,106 @@
     "build:cli": "webpack --config projects/xmacroscope-dvl-fw-plugin/src/lib/cli/webpack.config.js",
     "start-local": "npm run ng -- serve --configuration=local",
     "build-deploy": "npm run ng -- build --configuration=deploy --base-href=/mav/",
-    "build-demo": "npm run ng -- build --prod --base-href=/xmacroscope/",
-    "build-staging": "npm run ng -- build --prod --base-href=/"
+    "build-demo": "npm run ng -- build --configuration production --base-href=/xmacroscope/",
```

Comment:
isn't this supposed to be `--configuration=production` (not a space) ?

Reply:
Angular happily accepts both syntaxes. Also that was autogenerated during the angular migration/update

Reply:
Ok just checking